### ISSUE TITLE
Faster reopening and project resume

### ISSUE THREAD

**Issue by davidhigginson** [kooohei/brackets-synapse#73]
Would it be possible to make it possible to re-open the program and have re-opened files connect and begin syncing with remote server automatically?

**Comment by kooohei** [comment]
No Synapse is connected to server from auth when each command.
For example Save to the file, that try save to local file first, if it was succeed then that upload to the server.
When open the file, just a read file list if it was the directory, if it was file just download them.
create new file is little different, Synapse is create to the file on the local first, but that is not upload just a moment.
New file saved by us then upload to the server.

perhups, your problem was triggered when the take a time from connected to server. right?

Can you confirm to the log of the developer tools when It will happen?

I could not reproduce this problem.

if It is not problem.
please tell me the detail of your issue.

**Comment by Sentinel911** [comment]
davidhigginson って言うのは、発動する時に、Synapseのユーザセーブテータのような事をロードできませんのです。たとえ、セーブした様々のFTPファイルのルートを。今じゃ、コネクト時にファイルを再開するのは時間をかかるんです。

**Comment by kooohei** [comment]
ah I see, sorry I confused.
i.e Issue is "Would the tree view state return to before time", right?
I see, I thought that. and certainly it is possible.
but I think, this is wrong. because We don't know, who edit to the file on the remote server.
typically Brackets is installed to the local machine.
there is not changed filesystem by others.
but the remote server (specially web root) has shared context to others.

In conclusion.
will need more time when open the app with synapse, if the file system different  from before time.
because Synapse waited to the time of the "time out" sec, each different files.

**Comment by Sentinel911** [comment]
@kooohei it's fine, because I'm interested in implementation of  this feature as well.
Yes, we talk about saving user previous view state.
I suggest to do it as available option in settings, maybe by default turned off.
I understand the downside, but in my case I'm the only one with access to FTP so it will benefit my work without doubt.

**Comment by davidhigginson** [comment]
@Sentinel911 Thanks, that's exactly what I meant. I like the idea it being a setting which can be turned on and off. I, too, am the only person with FTP access, so I would benefit massively also.

**Comment by kooohei** [comment]
ok I promise to you, that append to extension at near future.
plz wait.

**Comment by davidhigginson** [comment]
@kooohei Its a great little extension already, and this is a really minor feature, so please put other, more important work first! Thank you ever so much!!

**Comment by Sentinel911** [comment]
koooheiさん、brackets-synapseを作ってくれましたどうもありがとうございます。素敵なお仕事でした。僕はbrackets-synapseを使っていますし、いいaddonと思いますよ。

**Comment by kooohei** [comment]
日本語でありがとうございます！

**Comment by JoeHana** [comment]
I + that request. I'm always the only one who works on the FTP and therefore it would benefit my workflow.

On another hand, I guess this would be even greater together with my other feature request in #74
